lib/jxl/enc_ans.cc

```diff
@@ -1569,33 +1588,49 @@ size_t WriteTokens(const std::vector<Token>& tokens,
   size_t numallbits = 0;
   // Writes in *reversed* order.
   auto addbits = [&](size_t bits, size_t nbits) {
-    JXL_DASSERT(bits >> nbits == 0);
-    if (JXL_UNLIKELY(numallbits + nbits > BitWriter::kMaxBitsPerCall)) {
-      out.push_back(allbits);
-      out_nbits.push_back(numallbits);
-      numallbits = allbits = 0;
+    if (nbits) {
```

Comment:
I assume this could have a JXL_UNLIKELY

Reply:
I'll add it, for lossless it doesn't seem to make much of a difference (since it's actually not really that unlikely), but for lossy I think it will help a little.